src/TestTask/components/CalculatorValue/CalculatorValue.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useRef, CSSProperties } from 'react'
+
+import { formatNumbers, FormatNumberThreshold, getScaleFromString, showPrefix } from '@consts/utils'
+
+import { Button, Grid, Input, Typography } from '@material-ui/core'
+
+import useStyles from './style'
+
+interface IProps {
+  value?: string
+  currency?: string
+  currencyIconSrc?: string
+  currencyRate: number
+  toCurrency: string
+}
+
+export const CalculatorValue: React.FC<IProps> = ({
+  value,
+  currency,
+  currencyIconSrc,
+  toCurrency,
+  currencyRate
+}) => {
+  const classes = useStyles()
+
+  return (
+    <Grid container className={classes.root}>
+      <Grid container className={classes.container}>
+        {/* INPUT VALUE */}
+        <Typography variant='h3' className={`${classes.title} ${!value && classes.titleEmpty}`}>{`${
+          value ? value : '0.0'
+        } ${currency === 'AERGO' ? 'ARG' : currency}`}</Typography>
```

Comment:
could you explain why are you using this 'AERGO'? I don't see anything like that on mockups

Reply:
The intention was to make the components more interactive, however it looks like it was unnecessary and could make no sense for those who are aware about bitcoins.